web/modules/custom/custom_page/config/install/core.entity_form_display.node.custom_page.collection_custom_page.yml

```diff
@@ -0,0 +1,52 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_form_mode.node.collection_custom_page
+    - field.field.node.custom_page.body
+    - field.field.node.custom_page.og_group_ref
+    - node.type.custom_page
+  module:
+    - path
+    - text
+id: node.custom_page.collection_custom_page
+targetEntityType: node
+bundle: custom_page
+mode: collection_custom_page
+content:
+  body:
+    type: text_textarea_with_summary
+    weight: 1
+    settings:
+      rows: 9
+      summary_rows: 3
+      placeholder: ''
+    third_party_settings: {  }
+  path:
+    type: path
+    weight: 4
+    settings: {  }
+    third_party_settings: {  }
+  promote:
+    type: boolean_checkbox
+    weight: 2
+    settings:
+      display_label: false
+    third_party_settings: {  }
+  sticky:
+    type: boolean_checkbox
+    settings:
+      display_label: true
+    weight: 3
+    third_party_settings: {  }
```

Comment:
Are all these options necessary? Is it mentioned in the functional specification that there should be fields visible for the paths, promote to frontpage and sticky content? I don't remember this being mentioned in the discussions. If this is offered to the end user then we should also make it work. At the moment we don't have a frontpage or stickied items yet so it's better to hide them now. If we need these functionalities we can enable it again in separate tickets.